bundles/rocky9.1/repo-templates/kubernetes.repo.gotmpl

```diff
@@ -0,0 +1,5 @@
+[kubernetes]
+name=Konvoy Kubernetes package repository
+baseurl=https://konvoy-packages-staging.s3.amazonaws.com/stable/linux/repos/el/kubernetes-v{{ .KubernetesVersion }}-{{ .RepoSuffix }}/x86_64
+gpgcheck=1
+gpgkey=https://konvoy-packages-staging.s3.amazonaws.com/stable/linux/repos/d2iq-sign-authority-gpg-public-key
```

Comment:
The old scripts used these staging repos because the build chain first pushed the packages there, then a different sync job would promote it to the `packages.d2iq.com` a few minutes later.
Here we can assume that the packages will already be published and want to match the repo that is used in KIB already. 
```suggestion
baseurl=https://packages.d2iq.com/konvoy/stable/linux/repos/el/kubernetes-v{{ .KubernetesVersion }}-{{ .RepoSuffix }}/x86_64
gpgcheck=1
gpgkey=https://packages.d2iq.com/konvoy/stable/linux/repos/d2iq-sign-authority-gpg-public-key
```

Reply:
awesome. didn't know that!